user-modules/common.nix

```diff
@@ -0,0 +1,11 @@
+{ ... }:
+{
+  # Common user modules that should always be imported
```

Comment:
move this file instead to the default.nix file. The default.nix can be imported by importing the directory which is closer to the convention of having a default set of imports. Remove this file afterwards.